test/templates/menu/index.njk

```diff
@@ -1,80 +1,25 @@
 {% extends "layout.njk" %}
 {% from "./../format.njk" import format %}
 {% from "./component.njk" import menu %}
+{% from "./component.njk" import menuWithAllHeadlines %}
```

Comment:
Not sure about Nunjucks syntax, but is there a way to import both `menu` and `menuWithAllHeadlines` from the file in one line?

Reply:
done